new/components/utils

```diff
@@ -39,10 +39,10 @@ function install_module
 
     if [[ ! -z "${venv}" ]]; then
         ctx logger info "Installing ${module} in virtualenv ${venv}..."
-        sudo ${venv}/bin/pip install ${module} >/dev/null
+        sudo ${venv}/bin/pip install ${module} --upgrade >/dev/null
```

Comment:
I don't want it to ignore installed. If a developer (or whoever) provided a branch to a module, it means they want to use that branch instead of the already installed one. This allows to upgrade that module.